### ISSUE TITLE
Incorrect conditional in lookback period logic

### ISSUE THREAD

**Issue by wlthomson** [openmsupply/mobile#2572]
## Describe the bug

Currently the lookback period logic has a typo/logic error causing the lookback period to be incorrectly applied to certain items. See https://github.com/openmsupply/mobile/pull/2564#issuecomment-603007520.

### To reproduce

Bug can be seen in the following code:

```
const addedRecently = itemAddedDate.isBefore(lookbackDate);
const startDate = amcEnforceLookback || !addedRecently ? lookbackDate : itemAddedDate;
```
### Expected behaviour

If not enforcing the lookback, the lookback is used if the item was added prior to the lookback date (currently using lookback if the item was added more recently than the lookback date).

### Proposed Solution

Small logic change:

```
const useLookbackDate = amcEnforceLookback || itemAddedDate.isBefore(lookbackDate);
const startDate = useLookbackDate ? lookbackDate : itemAddedDate;
```

### Version and device info

- App version: `v4.0.2`
- Tablet model: Android emulator.
- OS version: API 28.

### Additional context

N/A.

**Comment by wlthomson** [comment]
Fixed by #2607.

**Comment by wlthomson** [comment]
Confirmed fixed on `v5.0.0-rc3`.
